demo/site/src/redirects/cache.ts

```diff
@@ -0,0 +1,13 @@
+import { createCache, memoryStore } from "cache-manager";
+
+export const memoryCache = createCache(
+    memoryStore({
+        ttl: 15 * 60 * 1000, // 15 minutes,
+    }),
+    {
+        refreshThreshold: 5 * 60 * 1000, // refresh if less than 5 minutes TTL are remaining,
+        onBackgroundRefreshError: (error) => {
+            console.error("Error refreshing cache in background", error);
+        },
```

Comment:
If this is not set, the cache-manager would crash with a promise rejection. Wouldn't failing hard be the preferred scenario? Or does cache-manager try fetching the redirects again after an error?

Reply:
> Wouldn't failing hard be the preferred scenario?

no, that would bring down the whole node process (I suppose)

> Or does cache-manager try fetching the redirects again after an error?

I'm sure it will try a call in foreground